### DISCUSSION TITLE
Can't import component from typescript module in Next 14.1.3

### DISCUSSION THREAD

**Discussion by fb24m** [vercel/next.js#63047] [Help]
### Summary

I'm trying to import a component from a typescript library but next.js throws an error

Apparently there is a problem with ts, what exactly is wrong?

### Additional information
import:
```typescript
import { CubeLoader } from 'pixieui/components'
```
error:
![image](https://github.com/vercel/next.js/assets/103760918/927ba803-9cec-4deb-8d41-3c16b0f41258)

tsconfig - standard from Next
package - https://www.npmjs.com/package/pixieui

**Comment by icyJoseph** [comment]
Which TS version do you have? You most likely need to upgrade to one that supports the `import type {` syntax

Also, if you use VSCode, you need to make it use your project's TS version, and not its own. See how they do it here:

- https://nextjs.org/docs/app/building-your-application/configuring/typescript#typescript-plugin

**Comment by fb24m** [comment]
ok I got the answer on another forum
you need to add the line to next.config.js:
```typescript
transpilePackages: ['name of package(s)']
```
in my case it was necessary to specify
```typescript
transpilePackages: ['pixieui']
```
